src/api/helpers/helpers.js

```diff
@@ -0,0 +1,28 @@
+import cheerio from 'cheerio';
```

Comment:
This was a very non descriptive file name for one of the main functions in the app. Rename to `fetchAndTransformArticle` or something similar or better. Should probably be co-located with routes file.